python/batch-predictor/merlinpyspark/sink.py

```diff
@@ -44,10 +53,74 @@ def __init__(self, config: BigQuerySinkConfig):
         self._config = config
 
     def save(self, df: DataFrame):
-        df.write \
-            .mode(self._config.save_mode()) \
-            .format(self.WRITE_FORMAT) \
-            .option(self.OPTION_TABLE, self._config.table()) \
-            .option(self.OPTION_STAGING_BUCKET, self._config.staging_bucket()) \
-            .options(**self._config.options()) \
-            .save()
+        df.write.mode(self._config.save_mode()).format(self.WRITE_FORMAT).option(
+            self.OPTION_TABLE, self._config.table()
+        ).option(self.OPTION_STAGING_BUCKET, self._config.staging_bucket()).options(
+            **self._config.options()
+        ).save()
+
+
+class MaxComputeSink(Sink):
+    WRITE_FORMAT = "jdbc"
+    OPTION_URL = "url"
+    OPTION_DRIVER = "driver"
+    OPTION_QUERY_TIMEOUT = "queryTimeout"
+    OPTION_DB_TABLE = "dbtable"
+    OPTION_BATCH_SIZE = "batchSize"
+
+    def __init__(self, spark_session: SparkSession, config: MaxComputeSinkConfig):
+        self._config = config
+        self._spark = spark_session
+
+    def get_jdbc_url(self):
+        return f"jdbc:odps:{self._config.endpoint()}?project={self._config.project()}&accessId={self.get_access_id()}&accessKey={self.get_access_key()}&interactiveMode={self.get_interactive_mode()}&odpsNamespaceSchema=true&schema={self._config.schema()}&enableLimit=false"
+
+    def get_query_timeout(self):
+        return self._config.options().get("queryTimeout", "300")
+
+    def get_interactive_mode(self):
+        return self._config.options().get("interactiveMode", "true")    
+
+    def get_batch_size(self):
+        return self._config.options().get("batchSize", "10000")
+
+
+    def get_jdbc_driver(self):
+        return os.environ.get("ODPS_JDBC_DRIVER", "com.aliyun.odps.jdbc.OdpsDriver")
```

Comment:
I don't really know what ODPS JDBC is but does this mean we can potentially have other drivers to be configured? :o Similar question for the `ODPS_CUSTOM_DIALECT_CLASS` below. 

If we don't expect them to be anything else, should we just specify them as class constants?

Same questions for the `MaxComputeSource` class.

Reply:
Yes, we don't have a different driver now, but if for some reason we decide to bake / test a different jar we can use this env var to specify a different class.